ibazel/fswatcher/BUILD

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "factory_darwin.go",
+        "factory_fsnotify.go",
+    ],
+    importpath = "github.com/bazelbuild/bazel-watcher/ibazel/fswatcher",
+    visibility = ["//ibazel:__subpackages__"],
+    deps = [
+        "//ibazel/fswatcher/common:go_default_library",
+        "//ibazel/fswatcher/fsnotify:go_default_library",
+    	"//ibazel/log:go_default_library",        
```

Comment:
Nit: the indentation is off here as well.

Reply:
Thanks, my .editorconfig usually handles these but this project doesn't have one. it was a tab-spacing issue.